dashboard/test/models/script_test.rb

```diff
@@ -197,61 +132,6 @@ def populate_cache_and_disconnect_db
     assert_equal second_lesson_first_level, first_lesson_last_level.next_progression_level
   end
 
-  test 'blockly level in custom unit' do
-    game = Game.find_by_name('Studio')
-    create :level, name: 'blockly', game: game, level_num: 100
-    unit_data, _ = ScriptDSL.parse(
-      "lesson 'Lesson1', display_name: 'Lesson1'; level 'Level 1'; level 'blockly:Studio:100'", 'a filename'
-   )
-
-    unit = Script.add_unit({name: 'test script'}, unit_data[:lesson_groups])
-
-    assert_equal 'Studio', unit.script_levels[1].level.game.name
-    assert_equal '100', unit.script_levels[1].level.level_num
-  end
-
-  test 'allow applab and gamelab levels in unlaunched units' do
-    Script.add_unit(
-      {name: 'test script', published_state: SharedCourseConstants::PUBLISHED_STATE.beta},
-      [{
-        key: "my_key",
-        user_facing: true,
-        display_name: "Content",
-        lessons: [{name: "Lesson1", key: 'lesson1', script_levels: [{levels: [{name: create(:applab).name}]}]}]
-      }]
-    )
-    Script.add_unit(
-      {name: 'test script', published_state: SharedCourseConstants::PUBLISHED_STATE.beta},
-      [{
-        key: "my_key",
-        user_facing: true,
-        display_name: "Content",
-        lessons: [{name: "Lesson1", key: 'lesson1', script_levels: [{levels: [{name: create(:gamelab).name}]}]}]
-      }]
-    )
-  end
-
-  test 'allow applab and gamelab levels in login_required units' do
-    Script.add_unit(
-      {name: 'test script', published_state: SharedCourseConstants::PUBLISHED_STATE.preview, login_required: true},
-      [{
-        key: "my_key",
-        user_facing: true,
-        display_name: "Content",
-        lessons: [{name: "Lesson1", key: 'lesson1', script_levels: [{levels: [{name: create(:applab).name}]}]}]
-      }]
-    )
-    Script.add_unit(
-      {name: 'test script', published_state: SharedCourseConstants::PUBLISHED_STATE.preview, login_required: true},
-      [{
-        key: "my_key",
-        user_facing: true,
-        display_name: "Content",
-        lessons: [{name: "Lesson1", key: 'lesson1', script_levels: [{levels: [{name: create(:gamelab).name}]}]}]
-      }]
-    )
-  end
```

Comment:
the tests from line 200-253 seem to be covering seeding functionality, specifically the ability to add specific level types. I don't see any reason we'd need to test adding specific level types using our new lesson update API.